openshift_templates/performance_monitoring/pbench/pbench-agent-daemonset.yml

```diff
@@ -15,9 +15,9 @@ spec:
       hostPID: true
       hostNetwork: true
       containers:
-      - image: library/pbench-agent:latest
+      - image: docker.io/ravielluri/image:agent
```

Comment:
We should move test images to quay.io.  Not a blocker for this PR.   Will open an issue.

Reply:
Sounds good, will move the image to quay.io. Thanks for the review @mffiedler.